<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PrivateKeySelectorDialog</class>
 <widget class="QDialog" name="PrivateKeySelectorDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>321</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Choose an Existing Private Key</string>
  </property>
  <widget class="QListWidget" name="keyListWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>381</width>
     <height>221</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="passphraseLabel">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>240</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Passphrase:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="selectKeyButton">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>280</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Select Key</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>239</y>
     <width>251</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Password</enum>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
